doc/source/conf.py

```diff
@@ -67,6 +66,9 @@
 # CTSM-specific: version label used at the top of some pages.
 version_label = 'the latest development code'
 
+# List of versions to populate version picker dropdown menu
```

Comment:
Wait shouldn't this be "latest" and "clm5.0" now?

After the release we'll also add "ctsm6.0".

Reply:
Done with 7831b50.